src/test/resources/authority/fixedField.json

```diff
@@ -0,0 +1,31 @@
+{
+    "categoryCode": 1,
+    "headerTypeCode": 31,
+    "code": "008",
+    "displayValue": "",
+    "sequenceNumber": 0,
+		"dateEnteredOnFile":"180924",
```

Comment:
what's going on with the indentation here?

Reply:
I have changed it

Reply:
I have changed it